src/test/java/org/gridsuite/modification/server/modifications/AbstractNetworkModificationTest.java

```diff
@@ -95,7 +95,7 @@ public abstract class AbstractNetworkModificationTest {
     @Autowired
     protected ObjectMapper mapper;
 
-    private Network network;
+    protected Network network;
```

Comment:
Why ?
Just overload `getNetwork` method or just create a specific network in a test

Reply:
Done (setNetwork method added)